src/blueteam/Robot.java

```diff
@@ -34,6 +34,9 @@ void run() throws GameActionException {
 			checkHealth();
 			dodge();
 			step();
+			if (getRobotCount(RobotType.ARCHON) == 0)
+				if (rc.getTeamBullets() > 900)
+					rc.donate(rc.getVictoryPointCost());
```

Comment:
I would like to correct my previous statement. Let's leave it as it is in this pull request (not moving anything) and fix it in separate pull request. Maybe @hrnr  would like to make process of buying victory points little more inteligent.